test/e2e/flask/petnames-transactions.spec.js

```diff
@@ -37,6 +37,7 @@ describe('Petnames', function () {
         await switchToNotificationWindow(driver, 3);
         await expectName(driver, '0x0c54F...7AaFb', false);
         await saveName(driver, '0x0c54F...7AaFb', 'test.lens', undefined);
```

Comment:
@matthewwalsh0 wrote:
> In the signature tests, there were multiple addresses per confirmation, so for some we used a proposed name, and for one we saved a custom name.

> Here we are exclusively using a proposed name so there is an argument that we could have a separate test for saving a transaction with a custom name, though it's low risk since we know it's fundamentally the same underlying component.

> Equally we know that most transaction types ultimately use the sender-to-recipient component we've updated but for the sake of coverage, we could also add additional tests for different transaction types such as send transactions from within the wallet, or an approve transaction which uses some alternate components.